x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/build_sort_options.test.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { buildSortOptions } from './build_sort_options';
+
+describe('buildSortOptions', () => {
+  it('builds sort options from a list of field names', () => {
+    const sortOptions = buildSortOptions(
+      {
+        filterFields: [],
+        sortFields: ['fieldA', 'fieldB'],
+      },
+      [
+        {
+          name: 'Relevance ASC',
+          value: 'id',
+          direction: 'desc',
+        },
+        {
+          name: 'Relevance DESC',
+          value: 'id',
+          direction: 'asc',
+        },
+      ]
+    );
+
+    expect(sortOptions).toEqual([
+      {
+        name: 'Relevance ASC',
+        value: 'id',
+        direction: 'desc',
+      },
+      {
+        name: 'Relevance DESC',
```

Comment:
Sorry, I think I missed this - but why is this `ASC` (capitalized) vs `(asc)` in the later options? Should we update these static test examples to be consistent w/ each other? (mostly to reduce dev confusion)

Reply:
https://github.com/elastic/kibana/pull/88764/commits/ba1769cbe863866893a9116e9323cc70da853908